quantumdb-cli/src/main/java/io/quantumdb/cli/xml/XmlChangeset.java

```diff
@@ -11,6 +12,11 @@
 @Data
 public class XmlChangeset {
 
+	private final List<XmlOperation<?>> operations = Lists.newArrayList();
+	private String id;
+	private String author;
+	private String description;
+
```

Comment:
This moves a block of non-static fields above a static method. Try to the styling consistent: static stuff above non-static stuff, and fields above methods.

Reply:
Hmm, this is what the IntelliJ code style gives for arrangement, I imported your file:
![Code_Style_arrangement](https://user-images.githubusercontent.com/22129925/120387505-4c3c6000-c32a-11eb-94c3-d97d4a558323.png)